src/main/kotlin/no/nav/soknad/arkivering/soknadsfillager/repository/FilDbData.kt

```diff
@@ -1,13 +1,14 @@
 package no.nav.soknad.arkivering.soknadsfillager.repository
 
+import java.time.LocalDateTime
 import javax.persistence.Column
 import javax.persistence.Entity
 import javax.persistence.Id
 import javax.persistence.Table
 
 @Entity
 @Table(name = "documents")
```

Comment:
norks eller engelsk? vi er liksom midt mellom alt..

Reply:
Går for engelske navn på tabell og kolonner.